bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/dialogs/KontaktSelektor.java

```diff
@@ -571,6 +681,12 @@ public void run() {
 
 	}
 
+	private List<IContact> getMandantenKontakte() {
+		IQuery<IContact> query = CoreModelServiceHolder.get().getQuery(IContact.class);
+		query.and(ModelPackage.Literals.ICONTACT__MANDATOR, COMPARATOR.EQUALS, true);
```

Comment:
Stattdessen `IQuery<IMandator> query = CoreModelServiceHolder.get().getQuery(IMandator.class);` verwenden. Dann brauchts das query.and nicht